internal/engine/dockerprune/docker_pruner.go

```diff
@@ -35,57 +38,80 @@ func NewDockerPruner(dCli docker.Client) *DockerPruner {
 	return &DockerPruner{dCli: dCli}
 }
 
-func (dp *DockerPruner) DisableForTesting() {
-	dp.disabledForTesting = true
+func (dp *DockerPruner) DisabledForTesting(disabled bool) {
+	dp.disabledForTesting = disabled
 }
 
 func (dp *DockerPruner) OnChange(ctx context.Context, st store.RStore) {
 	if dp.disabledForTesting {
 		return
 	}
 
-	if dp.started {
+	if err := dp.sufficientVersionError(); err != nil {
```

Comment:
Should we log something in verbose in this case?

Reply:
will do